### ISSUE TITLE
GraphQL "data too long for column" error

### ISSUE THREAD

**Issue by justin0022** [tl-its-umich-edu/my-learning-analytics#954]
## Expected behavior (A clear and concise description of what you expected to happen) : 
User should be able to set as many custom assignment goals as they desire. 

## Describe the bug (Tell us what happens instead of the expected behavior) :
When user sets multiple custom assignment goals (during my testing, more than 3), the `default_view_value` can overflow as the user settings are saved as a JSON string. Likely this will need to be fixed on the DB end so that this column can store the long strings. 

## Steps to Reproduce :
1. Open AssignmentPlanningV2
2. Open devtools console
3. Set multiple custom grades (more than 3), and you'll see the error message below. 

## Screenshots/Video (If possible, add media to help explain your problem) :
<img width="1152" alt="Screen Shot 2020-06-04 at 3 43 03 PM" src="https://user-images.githubusercontent.com/8836578/83817626-1a8eab80-a67a-11ea-8c9e-dfe7bf26b0ff.png">

**Comment by ssciolla** [comment]
@justin0022, do you want help from someone with Django experience to help resolve this? I just want to make sure we have a plan; let me know if I can help.

**Comment by justin0022** [comment]
Sure, I'd appreciate your help in fixing this issue. I think changing the data type of the `default_view_value` column in `user_default_selection` table might be the fix.

**Comment by ssciolla** [comment]
@justin0022, I'm having trouble replicating this error. I entered in custom assignment goals (many of them long) for 17 assignments, and I did not encounter the error. My database utility (DBeaver) tells me that the field length for `user_default_selection.default_view_value` is `65535` characters; with what I entered, I only reached `1786`. Am I missing something? Or maybe the bug you encountered is for a slightly different issue?

**Comment by ssciolla** [comment]
@justin0022, I'm just following up on this. We're trying to release this week, so I'd like to figure out whether there's work that needs to be done on this.

**Comment by justin0022** [comment]
> @justin0022, I'm having trouble replicating this error. I entered in custom assignment goals (many of them long) for 17 assignments, and I did not encounter the error. My database utility (DBeaver) tells me that the field length for `user_default_selection.default_view_value` is `65535` characters; with what I entered, I only reached `1786`. Am I missing something? Or maybe the bug you encountered is for a slightly different issue?

Sorry just saw your reply. Hmm let me try again, I'll let you know shortly.

**Comment by justin0022** [comment]
Thank Sam, the DB migration solved this issue for me. My `default_view_value` was set to `varchar` before the migration and now it's set to `longtext`, so this issue is resolved.

**Comment by ssciolla** [comment]
Word! Glad it's not something something we have to worry about : )

**Comment by justin0022** [comment]
Same!
